sdk/eventhub/azure-eventhub/tests/perfstress_tests/receive_event.py

```diff
@@ -0,0 +1,39 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ._test_base import _ReceiveTest
+
+
+class ReceiveEventTest(_ReceiveTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        self._total_received_cnt = 0
+        self._async_total_received_cnt = 0
+
+    def _on_event_received(self, partition_context, event):
+        self._total_received_cnt += 1
+        if self._total_received_cnt > self.args.num_events:
```

Comment:
`>=`?
Not that I suppose one event makes much difference :D

Reply:
you're right! if it's an empty eh, then even one event might keeps the test from ending itself!
thanks to calling it out, updated!